### PR TITLE
kafkacat 1.3.1

### PR DESCRIPTION
**Pull request by yannick**
has a few new features (e.g. timestamps)

### CODE REVIEW

File: Formula/kafkacat.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@ class Kafkacat < Formula
   desc "Generic command-line non-JVM Apache Kafka producer and consumer"
   homepage "https://github.com/edenhill/kafkacat"
   url "https://github.com/edenhill/kafkacat.git",
-      :tag => "1.3.0",
-      :revision => "b79896cceedf81df7873850f38e5e2c4ad3e3e57"
-  revision 1
+      :tag => "1.3.1",
+      :revision => "0ab52045cde0c16f9f99dad41cee019ff88e974e"
+
```
Comment (neutric) [inline Formula/kafkacat.rb]: Please remove the above empty line.

**Review by neutric — CHANGES_REQUESTED** [review]

**Comment by neutric** [comment]
And while you are at it, please squash the commits and change the commit topic to `kafkacat 1.3.1`. Thanks!

**Comment by yannick** [comment]
@neutric done

**Comment by neutric** [comment]
@yannick Awesome, thanks for your contribution to Homebrew! 🍌

**Pull request closed without merging**